modules/setting/setting.go

```diff
@@ -705,18 +718,13 @@ func NewContext() {
 	if err = sec.MapTo(&LFS); err != nil {
 		log.Fatal("Failed to map LFS settings: %v", err)
 	}
-	LFS.ContentPath = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(AppDataPath, "lfs"))
-	if !filepath.IsAbs(LFS.ContentPath) {
-		LFS.ContentPath = filepath.Join(AppWorkPath, LFS.ContentPath)
-	}
+	LFS.ContentPath = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(appDataUserPath, "lfs"))
+	forcePathSeparator(LFS.ContentPath)
```

Comment:
Missing `LFS.ContentPath = `?

Reply:
The function do some validation instead of *force*:

```
func forcePathSeparator(path string) {
	if strings.Contains(path, "\\") {
		log.Fatal("Do not use '\\' or '\\\\' in paths, instead, please use '/' in all places")
	}
}
```

Maybe change the name to `mustPathSeparator`?